.npmignore

```diff
@@ -24,10 +24,10 @@ lighthouse-logger/
 # keep smokehouse tests, etc
 !lighthouse-cli/test
 
-node_modules/
+node_modules
```

Comment:
A trailing slash only ignore root folders. Omitting it will also ignore the deeper node_modules within in each docs.

fyi Published package doesn't have any issue here b/c we don't do `yarn install` for the docs in `lighthouse-pristine`.

Reply:
should we separate them from the others with a trailing `/` with a comment so it doesn't look like just mismatched formatting? :)